cmd/clusterctl/client/alpha/client.go

```diff
@@ -16,6 +16,12 @@ limitations under the License.
 
 package alpha
 
+import "context"
+
+var (
+	ctx = context.TODO()
```

Comment:
any plans to initialize a context here instead of using TODO()?

Reply:
Any reason not to add the context as an argument to the methods that are using it rather than having a global var? That would be the typical pattern for client calls like this. If there's a good reason for not doing now, but plans to later, I'd suggest adding a TODO comment that explains why this hasn't been done yet

Reply:
I followed the existing convention in [here](https://github.com/kubernetes-sigs/cluster-api/blob/master/cmd/clusterctl/client/cluster/client.go#L39). 

I don't have a strong opinion on this and I'm open to suggestions. I would like to keep the changes within the scope of this clean up PR, though.

Reply:
Looks like there needs to be a bigger effort across the codebase to sort these contexts out, I think it's ok in this case to leave as is and someone can come back to that at a later date, will defer to Cecile for decision on that